ansible/roles/monasca/templates/monasca-log-metrics/log-metrics.conf.j2

```diff
@@ -0,0 +1,59 @@
+# Generate metrics from logs. WIP
+
+input {
+    kafka {
+        zk_connect => "{{ monasca_zookeeper_servers }}"
+        topic_id => "{{ monasca_raw_logs_topic }}"
+        group_id => "transformer-logstash-consumer"
+        consumer_id => "monasca_log_metrics"
+        # This should be equal to the number of partitions
+        # https://www.elastic.co/guide/en/logstash/2.4/plugins-inputs-kafka.html#plugins-inputs-kafka-consumer_threads
+        consumer_threads => "4"
```

Comment:
kafka config has 1 partition. Doesn't sound like this  will be a big issue though.